package.json

```diff
@@ -8,13 +8,13 @@
     "npm": "9.*"
   },
   "scripts": {
-    "start": "BRANCH=local serverless offline start",
+    "start": "BRANCH=local SLS_DEBUG=* serverless offline start --noPrependStageInUrl",
```

Comment:
How come ya opted to put this in the start command and not in the Serverless.yml?

Reply:
copied it over from other repositories to see errors why the integration test is not running in github. can move to serverless.yaml if that would be better